transportable-udfs-examples/transportable-udfs-example-udfs/src/test/java/com/linkedin/transport/examples/TestFileLookupFunction.java

```diff
@@ -31,9 +32,13 @@ public void testFileLookup() {
     tester.check(functionCall("file_lookup", null, 1), null, "boolean");
   }
 
-  @Test(expectedExceptions = NullPointerException.class)
+  @Test
   public void testFileLookupFailNull() {
-    StdTester tester = getTester();
-    tester.check(functionCall("file_lookup", resource("file_lookup_function/sample"), null), null, "boolean");
+    try {
+      StdTester tester = getTester();
+      tester.check(functionCall("file_lookup", resource("file_lookup_function/sample"), null), null, "boolean");
+    } catch (NullPointerException ex) {
+      Assert.assertFalse(Boolean.valueOf(System.getProperty("trinoTest")));
+    }
   }
```

Comment:
The code of this test are executed for Trino, Hive and Spark. Originally it expects a NullPointerException is thrown during the execution of looking up a null value in a file by all three query engines. However, Trino v406 does not throw NullPointerException but returns a null value in this case. Therefore the code change removes the expected exception from @Test annotation, and only checks if a NullPointerException is caught if it is not in case of Trino. 
Also I have already added some comments to explain the reason in the code.